bonfire/Modules/Recycler/Views/listResource.php

```diff
@@ -1,16 +1,17 @@
 <?php $this->extend('master') ?>
 
 <?php $this->section('main') ?>
+
 <x-page-head>
     <div class="row">
         <div class="col">
             <h2>Recycler</h2>
         </div>
         <?php if(count($resources) > 1) : ?>
             <div class="col-auto">
-                <select name="resource" class="form-select">
+                <select name="r" class="form-select" hx-get="?" hx-target="#resource" hx-boost="true">
                 <?php foreach($resources as $alias => $details) : ?>
-                    <option value="<?= $alias ?>"><?= $details['label'] ?></option>
+                    <option value="<?= strtolower($alias) ?>" <?= ( isset($_GET) && isset($_GET['r']) && $_GET['r']===strtolower($alias)) ?  'selected' : ''?>><?= $details['label'] ?></option>
```

Comment:
Instead of checking `$_GET` directly here, I'd prefer to grab that value in the controller and pass it to the view. I think we already are, actually, with `currentAlias`.